### PR TITLE
Version bump and CHANGELOG updates in preparation for 5.0.3.

### PR DESCRIPTION
**Pull request by vkbo**
To be merged before tagging.
Open for corrections/additions.
If everything goes well, the plan is to tag on Monday.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,23 +4,27 @@
...
 * DYNK would not build with nagfor due to two malformed strings.
-* Fixed building with `ROND_ZERO` flag. The `matlib_bouncer` was trying to call non-existent round-towards-zero routines for `exp_mb()` functions (which is handled by round down) and `log10_mb` which was missing for `ROUND_ZERO` and has been added to `CRLIBM`.
-* A variable for DYNK FIR/IIR functions were written to memory twice, but for one case with an invalid index. This has been removed.
-* Removed remaining `COLLIMAT` flag in the beam-gas module that prevented the `BEAMGAS` flag from building.
+* Fixed building with `ROUND_ZERO` flag. The `matlib_bouncer` was trying to call non-existent round-towards-zero routines for `exp_mb()` functions (which is handled by round down `exp_rd()`), and `log10_mb` which was missing for `ROUND_ZERO` and has been added to `CRLIBM`.
+* A variable for DYNK FIR/IIR functions was written to memory twice, but with an invalid index for one of those writes. This write has been removed.
+* Removed a remaining `COLLIMAT` flag in the beam-gas module that prevented the `BEAMGAS` flag from building.
```
Comment (kyrsjo) [inline CHANGELOG.md]: "Removed a remaining `COLLIMAT` flag in the" -> "Removed a remaining `#IFDEF COLLIMAT` in the" ?

**Comment by rdemaria** [comment]
For some reason I cannot add this line in Build system part (github error 404)

* Added symlink name `sixtrack` pointing to the SixTrack executable in the build directory.

**Comment by vkbo** [comment]
Weird. I added it now. Thanks!

**Comment by rdemaria** [comment]
Would it make sense to add the PR number to the line in the change log?

**Pull request merged**